### ISSUE TITLE
improve error message during build when ToolProvider is unavailable

### ISSUE THREAD

**Issue by lbergelson** [broadinstitute/gatk#4532]
build.gradle finds the tool provider with the following line:

```
final javadocJDKFiles = files(((URLClassLoader) ToolProvider.getSystemToolClassLoader()).getURLs())
```
ToolPrivider.getSystemToolClassLoader() returns null on jre and certain other java installations.  This causes a confusing null pointer exceptions.

We should have a better error message when this happens.

**Comment by cmnbroad** [comment]
I was just adding the prerequisite checker for the large resources so I'll fix this while I'm at it.

**Comment by cmnbroad** [comment]
Fixed via #4530.
